### PR TITLE
1217 filter by status change

### PR DESCRIPTION
**Pull request by Dmac26**
﻿## Summary
Addresses Issue #1217  

This code update completes the 1217 card by updating all component.html, .ts, and .scss files that contribute to the flow, logic, and styling of the "Filter by" status text. Specifically, a switch statement was added to the .ts file to allow for specific messages to be displayed when user chooses a different condition to filter by and additional text was added to the "Filter by" label in the .html file. Styling was then adjusted to accommodate the other changes. 

## This pull request is ready to merge when...
- [x] Feature branch starts with the issue number
- [x] Is connected to its original issue via zenhub 👇
- [x] All tests are passing and meet coverage, linting, and accessibility requirements. And no security vulnerabilities ⚫️(Circle)
- [ ] This code has been reviewed by someone other than the original author

### CODE REVIEW

File: frontend/src/app/applications/permit-application-list/permit-application-list.component.ts [hunk 1]
```diff
@@ -44,6 +45,24 @@ export class PermitApplicationListComponent implements OnInit {
...
+      case 'rejected':
+        this.appStatusMessage = 'Applications that have been rejected for reasons provided by the permit administrator.';
+        break;
+      case 'cancelled':
+        this.appStatusMessage = 'Applications that have been submitted and subsequently cancelled by the applicant.';
+        break;
+      case 'expired':
+        this.appStatusMessage = 'Applications that have expired due to inactivity.';
+        break;
+    }
```
Comment (briandavidson) [inline frontend/src/app/applications/permit-application-list/permit-application-list.component.ts]: this switch looks good, i like the approach. only thing i'd add is i think we should add a `default` case as well.

File: frontend/src/sass/pages/_application-list.scss [hunk 2]
```diff
@@ -90,14 +90,17 @@ app-permit-application-list .usa-alert {
 }
 
 .application-status-toggle-wrapper {
-  max-width: 300px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
```
Comment (briandavidson) [inline frontend/src/sass/pages/_application-list.scss]: flexbox! 🔥

**Comment by briandavidson** [comment]
Oh, one thing I'm seeing now @Dmac26  -- don't forget to connect the PR with the issue.

File: frontend/src/sass/pages/_application-list.scss (hunk 2 above)
Comment (Dmac26) [reply frontend/src/sass/pages/_application-list.scss]: Yeah Buddy!

File: frontend/src/app/applications/permit-application-list/permit-application-list.component.ts (hunk 1 above)
Comment (Dmac26) [reply frontend/src/app/applications/permit-application-list/permit-application-list.component.ts]: Alrighty, I'll go ahead and add that default case.

**Pull request merged**